Source/Scene/Cesium3DTileStyle.js

```diff
@@ -236,6 +236,11 @@ define([
          * <p>
          * The expression must return or convert to a <code>Boolean</code>.
          * </p>
+         * <p>
+         * This expression is applicable to all tile formats.
+         * </p>
+         *
+         * Applicable to all tile formats.
```

Comment:
Remove this line, now a duplicate.

Reply:
woops - i must have missed that from the first time around. fixed now.